### PR TITLE
TRITON-2278 sdcnode builds for minimal-64-21.4.0

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -72,6 +73,18 @@ pipeline {
                     steps {
                         sh('''
 ./tools/build_jenkins -u 5417ab20-3156-11ea-8b19-2b66f5e7a439 -p $MIN_PLATFORM_STAMP
+        ''')
+                    }
+                }
+                stage('minimal-64-lts 21.4.0') {
+                    agent {
+                        node {
+                            label joyCommonLabels(image_ver: '19.4.0')
```
Comment (danmcd) [inline Jenkinsfile:82]: Copy-paste error, shouldn't this be 21.4.0?

Comment (bahamat) [reply Jenkinsfile:82]: Yep, and that invalidates my tests.

**Review by danmcd — DISMISSED** [review]
Much better.

**Review by danmcd — APPROVED** [review]
What does it take to revalidate?

**Comment by bahamat** [comment]
I just need the build artifacts from the Jenkins PR build and copy them onto a CN for testing.

**Review by danmcd — APPROVED** [review]

**Pull request merged**